timer/runtime/runtime.go

```diff
@@ -115,6 +115,13 @@ func (rt *TimerGroupRuntime) Start() {
 	go rt.loop()
 }
 
+// Running returns whether the runtime is running
+func (rt *TimerGroupRuntime) Running() bool {
```

Comment:
Is this function just used for test?

I'm thinking about using RWLock instead, if it is for test only, it is not necessary.

Reply:
it's only used by test now